phac-nml/irida-next
# [ENHC0010037] Member search component

## What does this PR do and why?
Moved the members search box into a new component.
No changes to the group & project member table sorting and searching functionality.
Depends on #657.

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/624f04b5-8fb5-4f0b-955a-eeb87cf8ac3c)
![image](https://github.com/user-attachments/assets/cc26fdbc-02e4-42e8-9862-ebcc86362567)

## How to set up and validate locally
1. Navigate to a project or group members page.
2. Use the search bar to filter by username.

See http://localhost:3000/rails/lookbook/inspect/search/default for preview.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
